pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -441,6 +441,13 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 								service.Annotations[awsLBSubnetsAnnotation] = JoinAWSSubnets(clbParams.Subnets, ",")
 							}
 						}
+
+						if eipAllocationsAWSEnabled {
+							nlbParams := getAWSNetworkLoadBalancerParametersInSpec(ci)
+							if nlbParams != nil && awsEIPAllocationsExist(nlbParams.EIPAllocations) {
+								service.Annotations[awsEIPAllocationsAnnotation] = ""
+							}
+						}
```

Comment:
I don't think this do what you want, the annotation isn't set by default anyways.

And this adds the annotation with a value of `""`, it's not clearing it. You should be able to just remove this.

Reply:
yes. i was already working on it. thanks for pointing out !